src/dialogxml/widgets/stack.cpp

```diff
@@ -55,25 +55,25 @@ void cStack::draw() {
 	for(auto& p : controls) {
 		p.second->draw();
 	}
-	if(drawFramed) drawFrame(2, frameStyle);
+	drawFrame(2, frameStyle);
```

Comment:
Did you figure out the bug you mentioned with the white lines appearing?

Reply:
I did not. I strongly suspect that this is the cause but I didn't get around to investigating in detail.